diff -uraN clonezilla-5.5.20/Makefile omv-clonezilla-5.5.20/Makefile
--- clonezilla-5.5.20/Makefile	2023-03-07 03:16:55.000000000 +0100
+++ omv-clonezilla-5.5.20/Makefile	2023-12-30 17:28:29.531243966 +0100
@@ -22,8 +22,8 @@
 
 install:
 	# install exec files
-	install -d ${DESTDIR}/usr/
-	cp -a sbin bin ${DESTDIR}/usr/
+	install -d ${DESTDIR}/usr/bib
+	cp -a sbin bin ${DESTDIR}/usr/bin/
 
 	# install setup dir
 	install -d $(DESTDIR)/$(SHAREDIR)/
